millipede/millipede.go

```diff
@@ -7,31 +7,51 @@
 package millipede
 
 import (
-	"strings"
+	"fmt"
 )
 
-type Millipede struct {
-	// Size is the amount of feet pairs
-	Size uint64
+type Pede string
+type Millipede []Pede
+
+func (m Millipede) Sprint() string {
+  str := ""
+
+  for _, pede := range m {
+    tmp := fmt.Sprintln(pede)
+    str = fmt.Sprint(str, tmp)
+  }
+
+  return str
+}
+
+func (m Millipede) Println() {
+  fmt.Print(m.Sprint())
+}
+
+// return a new Millipede Reversed
+// m.Reverse().Reverse().Reverse()
+func (m Millipede) Reverse() Millipede {
```

Comment:
This function reverse the lines, but don't changes the characters, in the python-version, they are using different characters for normal and reverse mode: https://github.com/getmillipede/millipede-python/blob/master/millipede/__init__.py#L44-L45

```json
        ...
        'default': {'bodyr': '╔═(███)═╗', 'body': '╚═(███)═╝',
                    'headr': '╔⊙ ⊙╗', 'head': '╚⊙ ⊙╝'},
        ...
```